openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -0,0 +1,2038 @@
+            'left': (10 + i * 40) + 'px',
+            'top': (110 + i * 40) + 'px'
+        });
+    });
+
+    $('#cover').remove();
+    $('#projectDef-spinner').hide();
+    $('#iri-spinner').hide();
+    $('#themeSelector').hide();
+    $('#saveBtn').prop('disabled', true);
+    $('#saveBtn').addClass('btn-disabled');
+
+    // Slider
+    $(function() {
+        $( '#slider-vertical' ).slider({
+            orientation: 'vertical',
+            range: 'min',
+            min: 0,
+            max: 100,
+            value: 16.666,
+            slide: function( event, ui ) {
+                $( '#econ-weight' ).val( ui.value );
+            }
+        });
+        $( '#econ-weight' ).val( $( '#slider-vertical' ).slider( 'value' ) );
+    });
+
+    $('#map-tools').append(
+        '<div class="l_proj"><button id="loadProjectdialogBtn" type="button" class="btn btn-blue">Load Project</button></div>'
+    );
+    // Using the list in order to keep the order
+    $.each(['indicators', 'svi', 'iri', 'projDef'], function(i, widgetName) {
+        // slice(1) removes the heading # from the selector
+        var buttonId = widgetsAndButtons[widgetName].button.slice(1);
+        var buttonText = widgetsAndButtons[widgetName].buttonText;
+        $('#map-tools').append(
+            '<div class="but_l_proj"><button id="' + buttonId + '" type="button" class="btn btn-blue" disabled>' + buttonText + '</button></div>'
+        );
+    });
+
+    if (webGl === false) {
+        $('#map-tools').append(
+            '<div><select id="leafletThematicSelection">'+
+                '<option>Select Indicator</option>'+
+            '</select><div>'
+        );
+
+        $('#leafletThematicSelection').hide();
+
+        setupLeafletMap();
+    } else {
+        $('#map-tools').append(
+            '<div><select id="webGlThematicSelection">'+
+                '<option>Select Indicator</option>'+
+            '</select></div>'
+        );
+
+        $('#webGlThematicSelection').hide();
+
+        setupMapboxGlMap();
+    }
+
+    $('#loadProjectdialogBtn').click(function() {
+        getGeoServerLayers();
+        $('#loadProjectDialog').dialog('open');
+    });
+
+    function toggleWidget(widgetAndBtn) {
+        // toggle widget and change text on the corresponding button
+        var btnText = $(widgetAndBtn.button).html();
+        // If the widget is visible, the button text is 'Hide Widgetname'
+        // Otherwise it is 'Show Widgetname'
+        // Let's change the button text before toggling the widget
+        if (btnText.indexOf('Hide ') >= 0) {  // Change Hide -> Show
+            btnText = 'Show ' + btnText.slice(5);
+            $(widgetAndBtn.button).css({
+                'background': '#f7931e'
+            });
+        } else {                              // Change Show -> Hide
+            btnText = 'Hide ' + btnText.slice(5);
+            $(widgetAndBtn.button).css({
+                'background': '#f05a24'
+            });
+        }
+
```

Comment:
Changed this path.
Before was: url: '/geoserver/oqplatform/ows?service=WFS&version=1.0.0&request=GetFeature&typeName='+ selectedLayer +'&outputFormat=json',